File: src/main/java/ar/edu/um/tesoreria/rest/extern/model/TituloEntregaFacultad.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.extern.model;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.OffsetDateTime;

import lombok.Data;

/**
 * @author daniel
 *
 */
@Data
public class TituloEntregaFacultad implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -216361084143866462L;

	private BigDecimal personaId;
	private Integer documentoId;
	private Integer facultadId;
	private Integer planId;
	private Integer carreraId;
	private Long tituloentregaId;
	private OffsetDateTime inicio;
	private OffsetDateTime entrega;
	private Long folio;
	private Integer libro;
	private String ministerio;
	private String formula;
	private String titulogrado;
	private String titulotrabajo;
	private BigDecimal personaIdasesor;
	private Integer documentoIdasesor;
	private OffsetDateTime fechaultima;
	private String materiaIdultima;
	private String libroultima;
	private Long folioultima;
	private Integer geograficaId;

}

File: src/main/java/um/tesoreria/rest/extern/model/kotlin/TituloEntregaFacultad.kt
package um.tesoreria.rest.extern.model.kotlin

import java.math.BigDecimal
import java.time.OffsetDateTime

data class TituloEntregaFacultad(

    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var facultadId: Int? = null,
    var planId: Int? = null,
    var carreraId: Int? = null,
    var tituloentregaId: Long? = null,
    var inicio: OffsetDateTime? = null,
    var entrega: OffsetDateTime? = null,
    var folio: Long? = null,
    var libro: Int? = null,
    var ministerio: String? = null,
    var formula: String? = null,
    var titulogrado: String? = null,
    var titulotrabajo: String? = null,
    var personaIdasesor: BigDecimal? = null,
    var documentoIdasesor: Int? = null,
    var fechaultima: OffsetDateTime? = null,
    var materiaIdultima: String? = null,
    var libroultima: String? = null,
    var folioultima: Long? = null,
    var geograficaId: Int? = null,

    )
